app/api/applications.js

```diff
@@ -74,9 +74,25 @@ async function processApplicationClaim (reference, user, approved) {
   }
 }
 
+async function getApplicationHistory (reference) {
+  const url = `${applicationApiUri}/application/history/${reference}`
+  try {
+    const response = await Wreck.get(url, { json: true })
+    if (response.res.statusCode !== 200) {
+      return { historyRecords: [] }
```

Comment:
maybe worth logging what happened here + statusCode

Reply:
I've updated the code and test, please see latest commit.